src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState: [],
+  reducers: {
+    addBook: (state) => [...state, { title: 'New Book' }],
```

Comment:
- [ ] Nice work 👍🏿 so far but as per project requirements you are to get the title and author as payload  from action creators.
 please implement it 
 
 
 ```javascript
addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
```